sdk/storage/tests.yml

```diff
@@ -10,29 +10,30 @@ jobs:
-        PRIMARY_STORAGE_ACCOUNT_KEY: $(java-storage-test-primary-account-key)
-        SECONDARY_STORAGE_ACCOUNT_NAME: $(java-storage-test-secondary-account-name)
-        SECONDARY_STORAGE_ACCOUNT_KEY: $(java-storage-test-secondary-account-key)
-        BLOB_STORAGE_ACCOUNT_NAME: $(java-storage-test-blob-account-name)
-        BLOB_STORAGE_ACCOUNT_KEY: $(java-storage-test-blob-account-key)
-        PREMIUM_STORAGE_ACCOUNT_NAME: $(java-storage-test-premium-account-name)
-        PREMIUM_STORAGE_ACCOUNT_KEY: $(java-storage-test-premium-account-key)
+        PRIMARY_STORAGE_ACCOUNT_NAME: $(PRIMARY_STORAGE_ACCOUNT_NAME)
+        PRIMARY_STORAGE_ACCOUNT_KEY: $(PRIMARY_STORAGE_ACCOUNT_KEY)
+        SECONDARY_STORAGE_ACCOUNT_NAME: $(SECONDARY_STORAGE_ACCOUNT_NAME)
+        SECONDARY_STORAGE_ACCOUNT_KEY: $(SECONDARY_STORAGE_ACCOUNT_KEY)
+        BLOB_STORAGE_ACCOUNT_NAME: $(BLOB_STORAGE_ACCOUNT_NAME)
+        BLOB_STORAGE_ACCOUNT_KEY: $(BLOB_STORAGE_ACCOUNT_KEY)
+        PREMIUM_STORAGE_ACCOUNT_NAME: $(PREMIUM_STORAGE_ACCOUNT_NAME)
+        PREMIUM_STORAGE_ACCOUNT_KEY: $(PREMIUM_STORAGE_ACCOUNT_KEY)
         AZURE_STORAGE_ACCESS_KEY: $(java-storage-test-access-key)
-        AZURE_STORAGE_FILE_ENDPOINT: $(java-storage-test-file-endpoint)
-        AZURE_STORAGE_FILE_CONNECTION_STRING: $(java-storage-test-file-connection-string)
-        AZURE_STORAGE_QUEUE_ENDPOINT: $(java-storage-test-queue-endpoint)
-        AZURE_STORAGE_QUEUE_CONNECTION_STRING: $(java-storage-test-queue-connection-string)
-        AZURE_STORAGE_BLOB_CONNECTION_STRING: $(java-storage-test-blob-connection-string)
-        STORAGE_DATA_LAKE_ACCOUNT_NAME: $(java-storage-data-lake-account-name)
-        STORAGE_DATA_LAKE_ACCOUNT_KEY: $(java-storage-data-lake-account-key)
-        AZURE_STORAGE_FILE_ACCOUNT_NAME: $(java-storage-account-name)
-        AZURE_STORAGE_FILE_ACCOUNT_KEY: $(java-storage-account-key)
-        AZURE_STORAGE_QUEUE_ACCOUNT_NAME: $(java-storage-account-name)
-        AZURE_STORAGE_QUEUE_ACCOUNT_KEY: $(java-storage-account-key)
-      TimeoutInMinutes: 120
+        AZURE_STORAGE_FILE_ENDPOINT: $(AZURE_STORAGE_FILE_ENDPOINT)
+        AZURE_STORAGE_FILE_CONNECTION_STRING: $(AZURE_STORAGE_FILE_CONNECTION_STRING)
+        AZURE_STORAGE_QUEUE_ENDPOINT: $(AZURE_STORAGE_QUEUE_ENDPOINT)
+        AZURE_STORAGE_QUEUE_CONNECTION_STRING: $(AZURE_STORAGE_QUEUE_CONNECTION_STRING)
+        AZURE_STORAGE_BLOB_CONNECTION_STRING: $(AZURE_STORAGE_BLOB_CONNECTION_STRING)
+        STORAGE_DATA_LAKE_ACCOUNT_NAME: $(STORAGE_DATA_LAKE_ACCOUNT_NAME)
+        STORAGE_DATA_LAKE_ACCOUNT_KEY: $(STORAGE_DATA_LAKE_ACCOUNT_KEY)
+        AZURE_STORAGE_FILE_ACCOUNT_NAME: $(AZURE_STORAGE_FILE_ACCOUNT_NAME)
+        AZURE_STORAGE_FILE_ACCOUNT_KEY: $(AZURE_STORAGE_FILE_ACCOUNT_KEY)
+        AZURE_STORAGE_QUEUE_ACCOUNT_NAME: $(AZURE_STORAGE_FILE_ACCOUNT_NAME)
+        AZURE_STORAGE_QUEUE_ACCOUNT_KEY: $(AZURE_STORAGE_QUEUE_ACCOUNT_KEY)
+        TimeoutInMinutes: 120
```

Comment:
This was turned into an EnvVar, before this was an override to the parent `yml` configuration of a 60 minute pipeline timeout. Either change the indent on this line and remove the newly added timeout or remove this line, both aren't needed.